libs/fssr/octree.h

```diff
@@ -183,8 +183,10 @@ class Octree
     int get_num_levels (Node const* node) const;
     void get_samples_per_level (std::vector<std::size_t>* stats,
         Node const* node, std::size_t level) const;
+    /* Pass root node center as parent node center at start of recursion. */
```

Comment:
Make this function private and provide a recusion start that passes the root node center. No explanation needed then.